src/components/sections/rich-text/themes/default/default.styles.ts

```diff
@@ -1,9 +1,61 @@
 'use client';
 
+import type { StyledSpacerProps } from './default.types';
+
+import { Heading } from '@/components/elements/heading';
 import { Link } from '@/components/elements/link';
-import { styled } from '@/styles';
+import { Text } from '@/components/elements/text';
+import { css, styled } from '@/styles';
+
 
 export const StyledLink = styled(Link)`
   text-decoration: underline;
   color: ${({ theme }) => theme.colors.text.darkGrey};
 `;
+
+export const StyledText = styled(Text)`
+  margin-bottom: 6px;
+`;
+
+export const StyledBiggerText = styled(Text)`
+  margin-bottom: 16px;
+`;
+
+export const StyledHeading = styled(Heading)`
+  margin-bottom: 24px;
+`;
+
+export const StyledSectionBreak = styled.div<StyledSpacerProps>`
+  ${({ $spacing }) => {
+    switch ($spacing) {
+      case 'policies':
+        return css`
+          margin-bottom: 26px;
+
+          @media (min-width: ${({ theme }) => theme.screens.lg}) {
+            margin-bottom: 42px;
+          }
+        `;
+      case 'docs':
+        return css`
+          margin-bottom: 40px;
+
+          @media (min-width: ${({ theme }) => theme.screens.lg}) {
+            margin-bottom: 64px;
+          }
+        `;
+      case 'simple':
+        return css`
+          margin-bottom: 24px;
+        `;
+      case 'normal':
+        return css`
+          margin-bottom: 48px;
+        `;
+      default:
+        return css`
+          margin-bottom: 48px;
+        `;
+    }
```

Comment:
@RBoraurito, I'm still unsure why we should have many spacing variants and a specific one for `policies`. Why not just have two:`content | blog`?

1. Content -> For all the rich pages like `Privacy policies` and `Terms of services`. For both of these pages, the spacing is `48px`, and it should look the same; if there is any inconsistency, we should ask the designer to fix it.
2. Blog -> For blog pages which is `64px`

Another alternative to avoid having this Spacer thing and force "editors" to add it every time they want to separate content is adding the `margin/padding` to each heading in the rich text. So, on blog pages, the heading will have a margin-top of 64px, and on regular rich text content pages, like terms of services, it will be 48px. So this way, the spacing will be automatically every time there is a heading between paragraphs. **_This option will also be beneficial for the current MD files content(Blogs/Docs), they will inherit automatically the spacing._**

Reply:
@RBoraurito, I think we should go with the heading `margin/padding` approach